src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -25,45 +21,78 @@ require Exporter;
 
 our @ISA = qw(Exporter);
 our @EXPORT = qw();
-our @EXPORT_OK = qw( make_figure_of_merit );
+our @EXPORT_OK = qw( make_figure_of_merit set_dynamic_mag_limits );
 our %EXPORT_TAGS = ( all => \@EXPORT_OK );
 
+use Inline Python => q{
+
+from chandra_aca.star_probs import acq_success_prob, prob_n_acq, mag_for_p_acq
 
-our $CUM_PROB_LIMIT = -5.2;
+def _acq_success_prob(date, t_ccd, mag, color, spoiler):
+    out = acq_success_prob(date, float(t_ccd), float(mag), float(color), spoiler)
+    return out.tolist()
 
+def _prob_n_acq(acq_probs):
+    n_acq_probs, n_or_fewer_probs = prob_n_acq(acq_probs)
+    return n_acq_probs.tolist(), n_or_fewer_probs.tolist()
+};
+
+our $CUM_PROB_LIMIT = 8e-3;
```

Comment:
> I haven't been able to find any catalogs that exceed it (as in A products) to have a good before/after test.

This is where we need to get outside of pure regression testing with available products.  You can modify a backstop file to effectively remove stars.  Because of the connectivity with other products (guide summary) you might get other errors popping but, but you could at least do one functional test (even if it doesn't get formally incorporated into testing).  I'm hoping to transition to starhopper within a year.